src/handlers/check_commits/force_push_range_diff.rs

```diff
@@ -54,10 +66,49 @@ pub(super) async fn handle_event(
     let (oldbase, oldhead) = (&compare_before.merge_base_commit.sha, before);
     let (newbase, newhead) = (&compare_after.merge_base_commit.sha, after);
 
-    // Rebase detected, post a comment to our range-diff.
-    event.issue.post_comment(&ctx.github,
-        &format!(r#"This PR was rebased onto a different {branch} commit! Check out the changes with our [`range-diff`]({protocol}://{host}/gh-range-diff/{org}/{repo}/{oldbase}..{oldhead}/{newbase}..{newhead})."#)
-    ).await.context("failed to post range-diff comment")?;
+    let message = format!(
+        r#"This PR was rebased onto a different {branch} commit! Check out the changes with our [`range-diff`]({protocol}://{host}/gh-range-diff/{org}/{repo}/{oldbase}..{oldhead}/{newbase}..{newhead})."#
+    );
+
+    // Rebase detected, post a comment linking to our range-diff.
+    post_new_comment(ctx, event, message).await?;
+
+    Ok(())
+}
+
+async fn post_new_comment(
```

Comment:
I don't like that this logic is here.

I would prefer if it was shared with the other check-commits sub-handlers, but given the requirement of only hiding the previous comment when posting a the new one, which is different from all the others sub-handler, it's was simpler to do it here rather than increasing the complexity in check-commits.

Reply:
I guess that in theory we could add the logic to the shared "check commits result" comment, but for now this is fine.